models/app.model.js

```diff
@@ -107,3 +107,17 @@ exports.findCommentByCommentId = (id) => {
       }
     });
 };
+
+exports.readAllUsers = () => {
+  return db
+    .query("SELECT * FROM users")
+    .then(({ rows }) => {
+      if (!rows.length) {
+        return Promise.reject({ status: 404, msg: "Users are not found" });
+      }
+      return rows;
+    })
+    .catch((err) => {
+      return Promise.reject(new Error("Server Error"));
+    });
```

Comment:
This is only here to pass the test that you wrote which is kind of why we don't suggest testing for a 500. 

The 500 error handler will do it's job, but trying to test for it is very tricky. In the case of a 'natural' 500 being thrown it would not end up in this catch block anyway and I would only keep catch blocks on your controllers as they are the end of the chain.